.github/CODEOWNERS

```diff
@@ -150,24 +150,31 @@
 # Management Plane
 # PRLabel: %Mgmt
 /sdk/advisor/arm-advisor @qiaozha @dw511214992 @MaryGao
+/sdk/advisor/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
```

Comment:
Can we avoid writing all these rules and do a wildcard rule like we do for api.md files?

```
# API review files
/sdk/**/review/*api.md @bterlson @xirzec @ramya-rao-a @jeremymeng
```

Reply:
I have updated the changes.

Reply:
Unfortunately wildcard syntax will not work for our purposes because some of our tools don't support it.

Reply:
Can we fix those tools? 😅

Reply:
We have talked about it before but decided it wasn't interesting enough to try and implement. However I do think we can probably add some basic support so I filed https://github.com/Azure/azure-sdk-tools/issues/2770 but that will not happen before want to get this change in.

Reply:
okay, @colawwj  Could you please revert this change?

Reply:
> Can we avoid writing all these rules and do a wildcard rule like we do for api.md files?
> 
> ```
> # API review files
> /sdk/**/review/*api.md @bterlson @xirzec @ramya-rao-a @jeremymeng
> ```

Implemented as of now. For details, please see:
- https://github.com/Azure/azure-sdk-tools/pull/5488
- https://github.com/Azure/azure-sdk-tools/issues/2770